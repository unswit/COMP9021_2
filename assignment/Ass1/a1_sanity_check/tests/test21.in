Please convert 49036 using fFeEdDcCbBaA
