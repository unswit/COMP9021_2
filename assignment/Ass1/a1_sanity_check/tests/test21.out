How can I help you? Sure! It is EeDEBBBaA
